docs/end-user-topics/ORCID.rst

```diff
@@ -0,0 +1,34 @@
+ORCIDs in Dockstore
+============================
+
+Have an ORCID_? You can now connect it to your Dockstore account.
+
+.. _ORCID: https://orcid.org/)
+
+Linking your ORCID to your Dockstore account
+--------------------------------------------
+
+The process is quite simple. If you are logged in now, you can `click here`_ to get there, or choose "Account" from the dropdown menu on the top right where your username is displayed. This will take you to a tab where you can control your linked services, such as Github and Quay. One of those, towards the bottom of the list, is the ability to link your account with ORCID.
+
+.. _click here: https://dockstore.org/accounts?tab=accounts
+
+.. image:: https://raw.githubusercontent.com/aofarrel/verbose-fiesta/master/Dockstore/Images/orcid_link_account_button.png
+  :alt: ORCID link account button
+
+From here, you'll be redirect to ORCID. With just a few clicks, you will now see this on the link services page of Dockstore.
+
+.. image:: https://raw.githubusercontent.com/aofarrel/verbose-fiesta/master/Dockstore/Images/orcid_after_account_linkage.png
```

Comment:
I think it would be better if you uploaded the images inside this repo instead of using your own. We've been putting images in this folder https://github.com/dockstore/dockstore-documentation/tree/develop/docs/assets/images/docs

Then you can do something like `.. image:: /assets/images/docs/image-name.png`